scripts/simulation/constructProposalCalldata.ts

```diff
@@ -31,7 +31,10 @@ function getTimelockCalldata(proposal: ExtendedAlphaProposal, proposalInfo: Prop
     combinedCalldatas.push(`${sighash}${proposal.calldatas[i].slice(2)}`);
   }
 
-  const salt = ethers.utils.id(proposalInfo.title);
+  const abiEncodedString = ethers.utils.defaultAbiCoder.encode(['string'], [proposalInfo.commands[0].description]);
```

Comment:
Why change to generate the salt from the description rather than the title?

Reply:
The solidity proposal doesn't have a title because it's just an array of actions.